test-app/yarn.lock

```diff
@@ -2574,6 +2579,14 @@ function-bind@^1.1.1:
   resolved "https://registry.yarnpkg.com/function-bind/-/function-bind-1.1.1.tgz#a56899d3ea3c9bab874bb9773b7c5ede92f4895d"
   integrity sha512-yIovAzMX49sF8Yl58fSCWJ5svSLuaibPxXQJFLmBObTuCr0Mf1KiPopGM9NiFjiYBCbfaa2Fh6breQ6ANVTI0A==
 
+functionless@^0.6.18:
```

Comment:
Should functionless be a peer dep for the language service or is this an uncommon consumption pattern?

Reply:
`functionless` is a peer dependency of `@functionless/language-service`